new/omnibus-chef-native-gems.md

```diff
@@ -0,0 +1,49 @@
+---
+RFC: unassigned
+Author: Lamont Granquist <lamont@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Title
+
+Omnibus Chef Native Gem Additions
+
+## Motivation
+
+    As a Chef User,
+    I want to have omnibus-chef ship with nokogiri,
+    Because that will speed up my initial converges, and ease my pain at installing it.
+
+    As a Chef User,
+    I want to have omnibus-chef ship with popular Open Source databases,
+    Because that will speed up my initial converaes, and again ease my pain.
+
+    As a Postgres Cookbook Author,
+    I want to obliterate what we had to do to install Postgres client into omnibus from a cookbook,
+    Because, seriously, have you look at that code?
```

Comment:
Yes, I helped write a lot of it, it is a confusing mess.  It is inherently awful to read and maintain and the hooks that it uses internally inside of ruby are not stable (broke in ruby-2.x that I had to fix) and the hooks that we used in the postgres build to force it to link against the omnibus openssl code is about as complex and brittle.  It isn't "bad code" in the sense that its not written by a "bad programmer", but it is not possible to clean up any better than it is, and requires specialized knowledge of the ruby library build system to understand and debug.  Making that code not need to exist WOULD make the world a better place, and we need to have a way to communicate that without people getting offended.